packages/server/package.json

```
@@ -30,10 +30,11 @@
     "access": "public"
   },
   "engines": {
-    "node": ">=14.0.0"
+    "node": ">=16.0.0"
   },
   "scripts": {
     "build": "rimraf dist && tsc",
+    "build:lerna-debug": "rimraf dist && tsc > output.txt; cat output.txt; rm output.txt",
```

This helped me troubleshoot `tsc` errors which Lerna was swallowing because it didn't output `stdout` on a non-0 exit code. See https://github.com/lerna/lerna/issues/3384